### PR TITLE
[10.0][IMP] Forward Ports

### PR DESCRIPTION
**Pull request by lasley**
This ports some changes over that have been merged into OCA/9.0

### CODE REVIEW

File: medical_family/__manifest__.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 {
 
     'name': 'Medical Family',
-    'version': '8.0.1.1.0',
+    'version': '9.0.1.0.0',
```
Comment (tedsalmon) [inline medical_family/__manifest__.py]: 10.0

File: medical_family/README.rst [hunk 2]
```diff
@@ -12,7 +13,7 @@ Usage
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/159/8.0
+   :target: https://runbot.odoo-community.org/runbot/159/9.0
```
Comment (tedsalmon) [inline medical_family/README.rst]: 10.0

File: sale_crm_medical_prescription/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": " Sale CRM - Medical Prescription",
+    "summary": "Create opportunities from prescriptions.",
+    "version": "9.0.1.0.0",
```
Comment (tedsalmon) [inline sale_crm_medical_prescription/__openerp__.py]: 10.0

**Review by tedsalmon — CHANGES_REQUESTED** [review]
There are tests failing. Some are because `napb_num` is still being referenced in views even after this field was changed to the proper name, `nabp_num`.

The other test failure is due to a broken assertion:
```
odoo.addons.medical_medicament.tests.test_medical_medicament: `   File "/home/travis/build/LasLabs/vertical-medical/medical_medicament/tests/test_medical_medicament.py", line 65, in test_active_component_ids
odoo.addons.medical_medicament.tests.test_medical_medicament: `     len(active), 1
odoo.addons.medical_medicament.tests.test_medical_medicament: ` AssertionError: 0 != 1
```

**Comment by woodbrettm** [comment]
@LasLabs/admins I performed a diff audit on this PR and the following was changed:

#### Dropped: Changes already present in release/10.0 (or better code)
* sale_crm_medical_prescription
* Convert all product_uom grams to units (except strength_uoms)

#### Moved
* medical_family port to: #195
* medical_prescription_us port to: #196

#### Current Changes Retained
* Missing medical_pharmacy_view and medical_medicament_view in sale_medical_prescription manifest
* Version bumps
* Making rx_line name readonly
* Some copyright stuff

**Comment by lasley** [comment]
I'm unclear as to the status of this PR review

**Review by tedsalmon — APPROVED** [review]

**Comment by tedsalmon** [comment]
Yeah, a lot has changed since I last looked at this, apparently. Anyways, LGTM!

**Pull request merged**